test/unit/pages/patientnew.test.js

```diff
@@ -108,6 +113,42 @@ describe('PatientNew', function () {
       expect(props.onUpdateDataDonationAccounts.callCount).to.equal(0);
       expect(props.trackMetric.callCount).to.equal(0);
     });
+    /* Regression test case to cover bug PT-304:
+        When a patient want to create his/her profile on Blip
+        If the local is 'fr' then the date format was incorrectly parsed
+    */
+    describe('When local=fr', () => {
+      before((done)=> {
+        i18n.off('languageChanged');
```

Comment:
I had to remove the listener on event "languageChanged" otherwise the method changeLanguage would not work...